apps/src/aiTutor/views/teacherDashboard/AccessControls.tsx

```diff
@@ -34,38 +38,80 @@ const AccessControls: React.FC<AccessControlsProps> = ({sectionId}) => {
     })();
   }, [sectionId]);
 
+  const columns = [
+    {
+      property: 'id',
+      header: {
+        label: 'Student ID',
+        props: {
+          style: {...tableStyles.headerCell, ...styleOverrides.headerCell},
+        },
+      },
+      cell: {
+        formatters: [(id: string) => <span>{id}</span>],
+        props: {style: {...tableStyles.cell, minWidth: '100px'}},
+      },
+    },
+    {
+      property: 'name',
+      header: {
+        label: 'Name',
+        props: {
+          style: {...tableStyles.headerCell, ...styleOverrides.headerCell},
+        },
+      },
+      cell: {
+        formatters: [(name: string) => <span>{name}</span>],
+        props: {style: {...tableStyles.cell, minWidth: '150px'}},
+      },
+    },
+    {
+      property: 'aiTutorAccessDenied',
+      header: {
+        label: 'AI Tutor Access',
+        props: {
+          style: {
+            ...tableStyles.headerCell,
+            ...styleOverrides.headerCell,
+            minWidth: '150px',
+          },
+        },
+      },
+      cell: {
+        formatters: [
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          (aiTutorAccessDenied: boolean, {rowData}: any) => (
```

Comment:
Why did we need to use `any` here?

Reply:
Because I was lazy when I got this working, and I didn't go back 😂 Nice catch!